docs/api/dialog.md

```diff
@@ -51,6 +51,8 @@ The `dialog` module has the following methods:
     untouched on Windows. For example, a button label of `Vie&w` will be
     converted to `Vie_w` on Linux and `View` on macOS and can be selected
     via `Alt-W` on Windows and Linux.
+  * `resolvesAliases` Boolean (optional) - Whether to resolve symlinks to their target path
```

Comment:
@MarshallOfSound #8623 was merged so there are some conflicts in this pull request now, let me know if you'd like me to resolve them.

If we do move this setting to `properties` it will probably need to have a negated name since it will be enabled by default for backwards-compatibility.

A couple name ideas for this:

- `noResolveAliases`
- `unresolvedAliases`